example/README.md

```diff
@@ -1,3 +1,96 @@
-# Example
+<picture>
```

Comment:
If you want a picture to be visible on pub.dev, then you need to use `![image]`

Reply:
it is required to use this tag in order to switch images for different github themes (dark and light)